144_Binary_Tree_Preorder_Traversal.py

```diff
@@ -0,0 +1,27 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+# Time complexy O(n)
+class Solution:
+    def preorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
+        ans = []
+        def loop(root, ans):
+            if root is None:
+                return ans
+            else:
```

Comment:
unnecessary

Reply:
modify loop function
- remove else in line 14
- remove a argment, ans